src/emm/emm_cli.py

```diff
@@ -211,5 +225,25 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_content_width=100, allow_extra_args=True))
+@click.option(
+    "--op",
+    "--operation",
+    type=click.Choice([a.value for a in GitAction]),
+    default=GitAction.CHECKOUT,
+    help="Git operations to perform with found commit [default=checkout].",
+)
+@click.option("-b", "--branch", default=None, help="Name of branch for git checkout.")
+@click.option("--directory", default=None, help="Directory to execute command at.")
+@click.pass_context
+def git(ctx: click.Context, op: str, branch: str, directory: str) -> None:
```

Comment:
I'm not sure we want to call this just `git` since the next piece of work we want to do here is also performing git operations. It might make more sense to call this something like `git-branch` or `branch`?